Tools/Scripts/libraries/reporelaypy/setup.py

```diff
@@ -20,19 +20,24 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from os.path import abspath, dirname, join
+from urllib.parse import urljoin
+from urllib.request import pathname2url
+
 from setuptools import setup
 
 
-def readme():
-    with open('README.md') as f:
-        return f.read()
+def _get_adjacent_package_requirement(name):
+    d = abspath(dirname(__file__))
+    package_path = join(d, "..", name)
+    file_url = urljoin("file://localhost", pathname2url(package_path))
+    return f"{name} @ {file_url}"
```

Comment:
Don't we only want to do this if the package is actually adjacent? I mention this because for `reporelaypy` and `resultsdbpy` in particular, it seems possible that future clients may genuinely install these packages from `pip`. In the past, there was a deployment of `resultsdbpy` that did just this.

Reply:
We should change this if/when we start publishing these packages to PyPI, IMO, potentially adjusting the names if we end up not having the current ones.

At the moment we practically don't expect the `setup.py` to ever be invoked (directly or indirectly), so this seems like a relatively non-issue.